jinja2/bccache.py

```diff
@@ -5,7 +5,7 @@
 
     This module implements the bytecode cache system Jinja is optionally
```

Comment:
While you’re in here, “that Jinja optionally uses” instead of “Jinja is optionally using”.